index.html

```diff
@@ -353,47 +353,43 @@ <h2>
         
         <div class="intro_2">
             <div class="headline">
-                <main>
-                    <h4><strong class="strong">About Myself</strong>
+                    <h2><strong class="strong">About Myself</strong>
```

Comment:
- [ ] The is marked as an `h2` however it's styling is not in confirmity with other `h2`s

**Other h2 on the page appear like this**
![h2](https://github.com/MayPyone/My_Portfolio/assets/99656687/ad31ad4a-d4ce-45a1-8db8-3392ec9c0b65)

**The About myself h2 appears like this**
![h2](https://github.com/MayPyone/My_Portfolio/assets/99656687/527158ac-7ab6-4b08-b1ae-f6194dda2fb4)